server/stripe/index.js

```diff
@@ -1,37 +1,73 @@
-require("dotenv").config()
-const stripe = require("stripe")(process.env.STRIPE_SECRET)
+require("dotenv").config();
+const stripe = require("stripe")(process.env.STRIPE_KEY);
+const decodeToken = require("../utils/decodeToken");
+const Profile = require("../models/Profile");
+const User = require("../models/User");
+const Payment = require("../models/Payment");
+
+// @route GET /payment/secret
+// @desc returns secret, create payment if there isn't one
+const getPaymentSecret = async (req, res, next) => {
+  let decoded = decodeToken(req.cookies.token);
+  const userId = decoded.id;
```

Comment:
You can obtain the user's id from the request object, we don't need to decode the token.

Reply:
I tried this in bookings controller, "POST" doesn't have `req.user.id`, "GET, PATCH, and DELETE" do